src/generateIncludes.js

```diff
@@ -13,6 +13,11 @@ export default function generateIncludes(simpleAST, type, context, options) {
   options = options || {};
 
   return Promise.all(Object.keys(simpleAST.fields).map(function (key) {
+    if (key === '__typename') {
```

Comment:
@mickhansen As far as I can tell from the spec, all other `__`vars start at the root of the document and are for introspection. Only `__typename` is used during queries themselves.